simulation_parameters/Constants.cs

```diff
@@ -1772,6 +1772,8 @@ public static class Constants
     public const float MIN_AUTO_EVO_STRENGTH_MULTIPLIER = 0.01f;
     public const float MAX_AUTO_EVO_STRENGTH_MULTIPLIER = 1.0f;
 
+    public const float HALF_BLINK_INTERVAL = 0.5f;
```

Comment:
I think this constant name is not clear enough what this is for...

Reply:
well I just moved this variable because it was said so in the TODO. I don't what it was for, I just kept the name

Reply:
Well the name should be prefixed with the class. I assume this is like from the patch node class? So the name should be something like `CURRENT_MAP_PATCH_INDICATOR_HALF_BLINK_INTERVAL`